docker/requirements.txt

```diff
@@ -35,6 +35,7 @@ tensorflow
 geojson
 
 # Geo stack
+geojson
```

Comment:
there is another `geojson` just 3 lines up

Reply:
Oops, it was supposed to be moved.